x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -21,14 +22,17 @@
 import org.elasticsearch.index.IndexSettings;
 import org.elasticsearch.indices.recovery.RecoveryState;
 import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.protocol.xpack.frozen.FreezeRequest;
 import org.elasticsearch.snapshots.SnapshotInfo;
 import org.elasticsearch.test.ESIntegTestCase;
+import org.elasticsearch.xpack.core.frozen.action.FreezeIndexAction;
+import org.elasticsearch.xpack.frozen.FrozenIndices;
```

Comment:
Similarly, not sure this is appropriate in terms of dependencies but I figured these interdependencies were ok for tests.

Reply:
For tests, it's ok I think

Reply:
I think it's OK too, but I'd like to avoid adding too many test conditionals to this test. Maybe we could test this in `AbstractSearchableSnapshotsRestTestCase` instead?

Reply:
Ok, I moved the tests to the REST suite.